.circleci/config.yml

```diff
@@ -284,6 +284,9 @@ jobs:
       - restore_cache: *restore_cache
       - run: *gh-config
       - run: *install
+      - run:
+          name: Install Lerna
+          command: sudo npm install -g lerna@3.13.1
```

Comment:
The scripts are running into access issues while trying to install this dependency. https://circleci.com/gh/forcedotcom/salesforcedx-vscode/9337

Reply:
I'd recommend not running `sudo` for npm install https://github.com/npm/npm/issues/2049#issuecomment-315665248
(remember `event-stream`?)